Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM gradle:6.1.1 as build
+
+WORKDIR /build
+
+COPY "gradle" "gradle"
+COPY "build.gradle" "settings.gradle" "gradle.properties" "/build"/
+
+RUN gradle build
+
+COPY src src
+
+RUN gradle build
+
+FROM openjdk:14
+
+WORKDIR /bot
+
+COPY --from=build /build/build/libs/bot-kt-*.jar /bot/
+
+RUN string1="$(ls bot-kt-*.jar)" && string2=${string1#"bot-kt-"} && echo ${string2%".jar"} > currentVersion
```

Comment:
does currentVersion exist in docker builds? it's a file that's autogenerated when the bot is run...

Reply:
Current version is created by the Dockerfile (when docker build is run)

Reply:
Sounds good